_studio/mfx_lib/cmrt_cross_platform/include/cmrt_cross_platform.h

```diff
@@ -49,6 +49,11 @@
 #endif
 #endif
 
+#if defined(ANDROID)
```

Comment:
I think this should be done both on Linux and Android, regardless. Moreover, on Linux we need to add -Wvirtual-dtor to force the check if it is not the default (I believe it is not, need to add here: builder/FindGlobals.cmake). In that way we will have aligned build between Linux/Android and avoid build regressions on Android. I believe such a change should be a separate preceding patch.

Reply:
done

Reply:
This apparently should be:
```
#if defined(__clang__)
  #pragma clang diagnostic push
  #pragma clang diagnostic ignored "-Wnon-virtual-dtor"
#elif defined(__GNUC__)
  #pragma GCC diagnostic push
  #pragma GCC diagnostic ignored "-Wnon-virtual-dtor"
#endif
```
This will eliminate the need of ANDROID define. And will not introduce issues on Windows (in case we add this).

Reply:
done